dbms/src/Common/TiFlashMetrics.h

```diff
@@ -395,10 +401,12 @@ namespace DB
     M(tiflash_fap_task_duration_seconds,                                                                                            \
       "",                                                                                                                           \
       Histogram,                                                                                                                    \
-      F(type_build_stage, {{"type", "build_stage"}}, ExpBuckets{0.05, 2, 60}),                                                      \
-      F(type_success, {{"type", "success"}}, ExpBuckets{0.05, 2, 60}),                                                              \
+      F(type_select_stage, {{"type", "select_stage"}}, ExpBuckets{0.1, 2, 120}),                                                    \
+      F(type_write_stage, {{"type", "write_stage"}}, ExpBuckets{0.05, 2, 60}),                                                      \
       F(type_ingest_stage, {{"type", "ingest_stage"}}, ExpBuckets{0.05, 2, 60}),                                                    \
-      F(type_total, {{"type", "total"}}, ExpBuckets{0.05, 2, 60}))                                                                  \
+      F(type_total, {{"type", "total"}}, ExpBuckets{0.5, 2, 200}),                                                                  \
+      F(type_queue_stage, {{"type", "queue_stage"}}, ExpBuckets{0.5, 2, 120}),                                                      \
+      F(type_phase1_total, {{"type", "phase1_total"}}, ExpBuckets{0.5, 2, 200}))                                                    \
```

Comment:
these `60/120/200` buckets size is too large for exp bucket size, this will lead to too many empty buckets in Prometheus. This lead to slow metrics fetching, more network flow waste on transferring metrics to prometheus and more stored size on prometheus.

I think 20 is a suitable bucket size for most "seconds" bucket, which have the upper bound as (base * 2^20) ~ base * 291 hours

Reply:
I will add ExpBucketsWithRange, and some static checks to prevent this error  from happening again